### ISSUE TITLE
How to add animation in Rmarkdown output ? 

### ISSUE THREAD

**Issue by cderv** [yihui/rmarkdown-cookbook#100]
I wonder it it could be interesting to takk about how to create animation in R and include them in web output. 

As there is now new 📦 efficient this, it could be an interesting thing to add

* [gganimate](https://stackoverflow.com/questions/30476119/plot-animation-in-knitr-rmarkdown) that uses [gifski](https://github.com/r-rust/gifski) but also other gif renderer (like av and magick)
* [av](https://ropensci.org/technotes/2018/10/06/av-release/) can also be a way to generate mp2 or gif  format
* magick can also convert to gif

## Related SO questions

* https://stackoverflow.com/questions/30476119/plot-animation-in-knitr-rmarkdown

**Comment by cderv** [comment]
To note that gifski has been a feature included in knitr, with a hook (also see #53 )
https://yihui.name/en/2018/08/gifski-knitr/

````r
```{r, animation.hook='gifski'}
for (i in 1:2) {
  pie(c(i %% 2, 6), col = c('red', 'yellow'), labels = NA)
}
```
````

**Comment by yihui** [comment]
https://bookdown.org/yihui/rmarkdown-cookbook/animation.html
